docs/model/simulation_outputs.rst

```diff
@@ -377,6 +386,9 @@ An example of the created energy system model graphs is shown below.
 .. image:: images/network_graph.png
  :width: 600
 
+Files
+#####
+
```

Comment:
We should rather rename `Figures` to `Files`, as all of them are enumerated as files together at the top of the page:
![grafik](https://user-images.githubusercontent.com/44204527/117445401-eb905200-af3a-11eb-942a-a1853637b2ec.png)

Also, I noticed that we should add "files" to this sentence:
![grafik](https://user-images.githubusercontent.com/44204527/117445447-f9de6e00-af3a-11eb-86fc-4ed4f92c647e.png)

(If you want to distinguish into files and figures, we need to introduce a new kpi kategory in `mvs_kpi_categories.csv`

Reply:
Do you mean we should use both of the categories, ie. `files` and `figures`?